tasks.py

```diff
@@ -162,6 +162,15 @@ def phpstan(c):
         docker_compose_run(c, 'php ./vendor/bin/phpstan analyse')
 
 
+@task
+def ngrok(c):
+    """
+    Open the local web server with Ngrok
+    """
+    with Builder(c):
+        c.run('./ngrok http -host-header=local.starfleet.app local.starfleet.app:443')
```

Comment:
./ngrok does not exist on my computer. usually people install it globally, so you can use:

```suggestion
        c.run('ngrok http -host-header=local.starfleet.app local.starfleet.app:443')
```